src/MSfMData.hpp

```diff
@@ -91,7 +91,7 @@ class MSfMData : public QObject
     inline int nbCameras() const {
         if(!_sfmData || _status != Ready)
             return 0;
-        return _sfmData->getValidViews().size();
+        return (int)_sfmData->getValidViews().size();
```

Comment:
Would it make more sense to return a std::size_t instead of int? I don't know if there are other constraints for using `int` but it seems more natural.

Reply:
Done! I think the `int` was there because it didn't require to be registered with Qt to be used in properties.